src/api.jl

```diff
@@ -134,12 +134,12 @@ function infer(ra_range::Tuple{Float64, Float64},
     # Get indicies of entries in the  RA/Dec range of interest.
     entry_in_range = entry->((ra_range[1] < entry.pos[1] < ra_range[2]) &&
```

Comment:
What will happen if you choose an (RA, DEC) rectangle in
query_overlapping_fieldids that contains the edge of a frame?  This will
not work correctly, right?

I know this code didn't change with this PR, but unfortunately I didn't
look over the original code carefully, so I'm just pointing out problems
now as I find them.

On Tue, Mar 22, 2016 at 4:30 PM, Kyle Barbary <notifications@github.com>
wrote:

> In src/api.jl
> <https://github.com/jeff-regier/Celeste.jl/pull/174#discussion_r57087980>:
>
> > @@ -134,12 +134,12 @@ function infer(ra_range::Tuple{Float64, Float64},
> >      # Get indicies of entries in the  RA/Dec range of interest.
> >      entry_in_range = entry->((ra_range[1] < entry.pos[1] < ra_range[2]) &&
>
> I think we would want to add the border when we call query_overlapping_fieldids(ramin,
> ramax, decmin, decmax) in infer_nersc. The find(entry_in_range, catalog)
> should only get sources in the target sky patch.
>
> BTW, this code hasn't changed in this PR - this was just a variable rename
> for consistency.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/jeff-regier/Celeste.jl/pull/174/files/249556c8da20320ca889e1dfa8c6faef40973454#r57087980>
>